src/test/java/io/debezium/connector/spanner/AbstractSpannerConnectorIT.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.spanner;
+
+import org.junit.jupiter.api.AfterAll;
+
+import io.debezium.config.Configuration;
+import io.debezium.connector.spanner.util.Connection;
+import io.debezium.connector.spanner.util.Database;
+import io.debezium.connector.spanner.util.KafkaEnvironment;
+import io.debezium.embedded.AbstractConnectorTest;
+
+public class AbstractSpannerConnectorIT extends AbstractConnectorTest {
+
+    private static final KafkaEnvironment KAFKA_ENVIRONMENT = new KafkaEnvironment(
+            KafkaEnvironment.DOCKER_COMPOSE_FILE);
+    protected static final Database database = Database.TEST_DATABASE;
+    protected static final Connection databaseConnection = database.getConnection();
+    protected static final int WAIT_FOR_CDC = 1000;
```

Comment:
Integration tests are always tricky WRT timeouts. Could you please introduce something like `io.debezium.connector.sqlserver.util.TestHelper.waitTimeForRecords()`?

Reply:
Agree, I introduced a similar `waitTimeForRecords` function with value of 3 seconds. Instead of the previous `Thread.sleep(x millisconds)` to wait to records, I changed them to the existing test util function` waitForAvailableRecords(waitTimeForRecords(), TimeUnit.SECONDS);`